includes/compatibility/class-compat.php

```diff
@@ -108,5 +113,11 @@ public function _fix_pagination_issue( $clauses, $query ) {
 
 		return $clauses;
 	}
-
+	public function wp_fusion_skip_auto_login( $skip_auto_login ) {
```

Comment:
Let's add a doc block in here. Don't forget to leave a blank line between functions too.